index.js

```diff
@@ -88,6 +88,13 @@
   //  identity :: a -> a
   var identity = function(x) { return x; };
 
+  // constant :: a -> b -> a
+  var constant = function(x) {
```

Comment:
Please add one more space between `//` and `constant` to align the function name in type line with the identifier on the following line. :)

Also, for lack of a better system the helper functions are defined in alphabetical order, so `constant` should be defined before `identity`.